src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,46 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
+  <>
+    <div className="card" data-cy="Movie">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <img
+            data-cy="MovieImage"
+            src={movie.imgUrl}
```

Comment:
Pls use destructuring for `movie` object

Reply:
![tests](https://github.com/mate-academy/react_movies-list-js/assets/73246166/9d14ef9f-3b07-474e-a07a-7d67c88a2034)

Reply:
Cannot did this, cause tests